eq-author/src/components/EditorLayout/Header/PageTitle.js

```diff
@@ -31,7 +31,7 @@ const PageTitle = ({ children }) => (
 );
 
 PageTitle.propTypes = {
-  children: PropTypes.node.isRequired,
+  children: PropTypes.node,
```

Comment:
Great! this will fix a consol error for commentsPanel too :)

Reply:
Yes, it was quite annoying 😁

Reply:
Skillz